.github/workflows/release.yml

```diff
@@ -8,8 +8,7 @@ env:
   BUCKET_NAME: optic-packages
   PACKAGE_NAME: api
   NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
what do you think about setting this on the steps that need it vs. setting global env?

Reply:
🤷‍♂️ indifferent!